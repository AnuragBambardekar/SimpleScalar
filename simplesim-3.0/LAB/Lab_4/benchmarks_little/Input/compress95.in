25000        e 2231

