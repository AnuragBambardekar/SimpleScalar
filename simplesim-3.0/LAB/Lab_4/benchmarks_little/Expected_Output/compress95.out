SPEC 129.compress harness
Initial File Size:25000	Start character:e
The starting size is: 25002
The compressed size is: 15195
The compressed/uncompressed size is: 25002
Files both have length 25002
First character (r) and Last Character (
) match. 
The starting size is: 25005
The compressed size is: 15196
The compressed/uncompressed size is: 25005
Files both have length 25005
First character (r) and Last Character (
) match. 
The starting size is: 25009
The compressed size is: 15200
The compressed/uncompressed size is: 25009
Files both have length 25009
First character (r) and Last Character (
) match. 
The starting size is: 25014
The compressed size is: 15202
The compressed/uncompressed size is: 25014
Files both have length 25014
First character (r) and Last Character (
) match. 
The starting size is: 25020
The compressed size is: 15205
The compressed/uncompressed size is: 25020
Files both have length 25020
First character (r) and Last Character (
) match. 
The starting size is: 25027
The compressed size is: 15209
The compressed/uncompressed size is: 25027
Files both have length 25027
First character (r) and Last Character (
) match. 
The starting size is: 25035
The compressed size is: 15212
The compressed/uncompressed size is: 25035
Files both have length 25035
First character (r) and Last Character (
) match. 
The starting size is: 25044
The compressed size is: 15216
The compressed/uncompressed size is: 25044
Files both have length 25044
First character (r) and Last Character (
) match. 
The starting size is: 25054
The compressed size is: 15219
The compressed/uncompressed size is: 25054
Files both have length 25054
First character (r) and Last Character (
) match. 
The starting size is: 25065
The compressed size is: 15223
The compressed/uncompressed size is: 25065
Files both have length 25065
First character (r) and Last Character (
) match. 
The starting size is: 25077
The compressed size is: 15226
The compressed/uncompressed size is: 25077
Files both have length 25077
First character (r) and Last Character (
) match. 
The starting size is: 25090
The compressed size is: 15230
The compressed/uncompressed size is: 25090
Files both have length 25090
First character (r) and Last Character (
) match. 
The starting size is: 25104
The compressed size is: 15233
The compressed/uncompressed size is: 25104
Files both have length 25104
First character (r) and Last Character (
) match. 
The starting size is: 25119
The compressed size is: 15237
The compressed/uncompressed size is: 25119
Files both have length 25119
First character (r) and Last Character (
) match. 
The starting size is: 25135
The compressed size is: 15240
The compressed/uncompressed size is: 25135
Files both have length 25135
First character (r) and Last Character (
) match. 
The starting size is: 25152
The compressed size is: 15244
The compressed/uncompressed size is: 25152
Files both have length 25152
First character (r) and Last Character (
) match. 
The starting size is: 25170
The compressed size is: 15247
The compressed/uncompressed size is: 25170
Files both have length 25170
First character (r) and Last Character (
) match. 
The starting size is: 25189
The compressed size is: 15251
The compressed/uncompressed size is: 25189
Files both have length 25189
First character (r) and Last Character (
) match. 
The starting size is: 25209
The compressed size is: 15254
The compressed/uncompressed size is: 25209
Files both have length 25209
First character (r) and Last Character (
) match. 
The starting size is: 25230
The compressed size is: 15258
The compressed/uncompressed size is: 25230
Files both have length 25230
First character (r) and Last Character (
) match. 
The starting size is: 25252
The compressed size is: 15261
The compressed/uncompressed size is: 25252
Files both have length 25252
First character (r) and Last Character (
) match. 
The starting size is: 25275
The compressed size is: 15265
The compressed/uncompressed size is: 25275
Files both have length 25275
First character (r) and Last Character (
) match. 
The starting size is: 25299
The compressed size is: 15268
The compressed/uncompressed size is: 25299
Files both have length 25299
First character (r) and Last Character (
) match. 
The starting size is: 25324
The compressed size is: 15272
The compressed/uncompressed size is: 25324
Files both have length 25324
First character (r) and Last Character (
) match. 
The starting size is: 25350
The compressed size is: 15275
The compressed/uncompressed size is: 25350
Files both have length 25350
First character (r) and Last Character (
) match. 
